### ISSUE TITLE
기본 코드 에러 관련 질문 드립니다

### ISSUE THREAD

**Issue by kwonmha** [naver/nlp-challenge#23]
1. 기본으로 주어지는 코드를 보면 model.py의 sparse_cross_entropy_loss 함수에서 self.logits, self.targets, self.mask, self.Y, self.S 이런 변수들이 정의되어 있지 않습니다.
이 변수들은 nsml 상에서 모델을 실행하면 자동으로 설정되는 변수들인가요?

2. tensorflow 버전 문제
주로 tensorflow 1.11과 cuda 9.0을 사용하다 보니 가상환경을 만들어서 실제 nsml 도커에서 사용되는 tensorflow 1.4.1 버전으로 main.py를 실행시키면 에러가 나면서 사용 불가능합니다. tf 1.4.1에서 cudao 9.0을 사용 가능하게 한 wheel이 있어서 해봤는데 역시 오류가 나고요.
그리고 setup.py의 tensorflow 버전을 1.11로 바꾸는 것도 안 되는 것 같네요. 
사용되는 코드의 기준을 1.4.1로 맞춰야 하나요?

**Comment by nsml-admin** [comment]
안녕하세요~

1. 해당하는 함수와 변수들은 실제 baseline이 돌아가는동안에는 안쓰이는 함수입니다. 지워서 baseline에 반영하였습니다. 감사합니다.

2. 코드의 기준을 꼭 1.4.1로 맞추실 필요는 없습니다. baseline이 실행되는 환경이 1.4.1에만 맞춰져있고 실제 submit할 때의 버전 제한은 없습니다.

감사합니다!
